src/interfaces/api/site.ts

```diff
@@ -546,6 +550,10 @@ export class GetSiteResponse {
   @Type(() => Language)
   all_languages: Language[];
   discussion_languages: number[];
+  @Transform(({ value }) => toOption(value), { toClassOnly: true })
+  @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
+  @Expose()
```

Comment:
You will probably need to add the type here, coulda swore you had that with the other one.

Reply:
Added it. I'm not sure what those do, though, just following convention lol